### PR TITLE
feat (login) ✨ Build login UI

### PR DESCRIPTION
**Pull request by asmashawky**
## Description 📖 
The login screen will allow the user to log in for the app using his  email and password

## How to Test 📋 
Just open the signup screen and enter an email and password.
## Screenshot /Recordings 
photo_٢٠٢٤-٠٩-٣٠_١٩-٥٢-٥٩

[DIVA-53]: https://aasemhany.atlassian.net/browse/DIVA-53?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: lib/core/widgets/my_Text_Button.dart [hunk 1]
```diff
@@ -0,0 +1,58 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_screenutil/flutter_screenutil.dart';
+
+import '../themes/colors.dart';
+
+class myTextButtom extends StatelessWidget {
```
Comment (omarahmedx14) [inline lib/core/widgets/my_Text_Button.dart]: please rename it to MyTextButtom

File: lib/core/widgets/my_Text_Feild.dart [hunk 2]
```diff
@@ -0,0 +1,53 @@
+import 'package:diva/core/themes/colors.dart';
+import 'package:flutter/material.dart';
+
+import '../themes/styles.dart';
+
+class mytextfeild extends StatelessWidget {
```
Comment (omarahmedx14) [inline lib/core/widgets/my_Text_Feild.dart]: please rename it to MyTextField

File: lib/features/login/ui/widgets/dont_Have_Account.dart [hunk 3]
```diff
@@ -0,0 +1,28 @@
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/gestures.dart';
+
+import '../../../../core/themes/styles.dart';
+
+class dontHaveAccount extends StatelessWidget {
```
Comment (omarahmedx14) [inline lib/features/login/ui/widgets/dont_Have_Account.dart:6]: please rename it to DontHaveAccount

**Review by omarahmedx14 — CHANGES_REQUESTED** [review]

File: lib/features/login/ui/widgets/dont_Have_Account.dart (hunk 3 above)
Comment (asmashawky) [reply lib/features/login/ui/widgets/dont_Have_Account.dart:6]: RENAMED PLZ ACCEPT IT YA OMAR 🥹 🥹

File: lib/core/widgets/my_Text_Feild.dart (hunk 2 above)
Comment (asmashawky) [reply lib/core/widgets/my_Text_Feild.dart]: RENAMED PLEEEEZ ACCEPT YA OMAR 👯

File: lib/core/widgets/my_Text_Button.dart (hunk 1 above)
Comment (asmashawky) [reply lib/core/widgets/my_Text_Button.dart]: RENAMED PLEEEEZ ACCEPT YA OMAR 🧛

**Review by omarahmedx14 — APPROVED** [review]

**Pull request merged**